Integrations/integration-rasterize.yml

```diff
@@ -41,7 +41,7 @@ script:
             friendlyName = 'email.pdf'
 
         f = open('htmlBody.html','w')
-        f.write(html)
+        f.write('<html style="background:white";>' + html + '</html>')
```

Comment:
I think it is not related, try to checkout from master

Reply:
no idea why is this here. will fix